static/app/views/organizationStats/index.tsx

```diff
@@ -268,9 +329,15 @@ export class OrganizationStats extends Component<Props> {
     const {organization} = this.props;
     const hasTeamInsights = organization.features.includes('team-insights');
 
+    // We only show UsageProjectStats if multiple projects are selected
+    const shouldRenderProjectStats = this.hasProjectStats
```

Comment:
🤔 
personally i dislike this.
It's unexpected to have the page perform so differently while i'm still on teh same page/route/tab...

I would prefer to go back to the drawing board wtih this and figure out a better flow :(

Reply:
I still prefer this to having pages show similar information but be accessible through different routes. E.g. If we are keeping org stats around, but showing that same graph somewhere else like `stats/project/<project-slug>/` and push the user to that new route.

Then we have almost the same UI served from two different areas in the app, and we have to worry about how to direct users to one versus the other, and navigating between them.
I personally don't see the harm since we can surface the differences elsewhere in the app by linking directly to `stats/?project=1` or adding placeholder boxes which say `To view [X] data, select a single project`.

cc @jmanhart Curious to hear your thoughts on this design as well!